nlp/src/main.py

```diff
@@ -369,17 +388,44 @@ def process_fn(text):
     ).json()
 
 
-# TODO: Implement troll model
-@app.post("/troll")
-async def get_troll(request: TextRequest):
+@app.post("/ai-detector", response_model=TextResponse,

```

Comment:
Maybe we could rename that to `llm-detection`? This would fit in better with already existing analysis names.